hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1926,7 +1926,6 @@ public void stop() {
       if (omSnapshotProvider != null) {
         omSnapshotProvider.stop();
       }
-      omState = State.STOPPED;
```

Comment:
If we remove state transition, `stop()` may perform its actions multiple times.  I think a better way to ensure system exit would be moving the `exitSystem()` call outside of the `if` in `shutdown` (both methods):

https://github.com/apache/ozone/blob/e6179c1ec6d77a46687c513ab63b81d7c62e9796/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java#L782-L794

Reply:
it is a duplicated `omState = State.STOPPED;` in `ozoneManager#stop`

Reply:
@JacksonYao287 Thanks for the info, I didn't notice the other state change at the start of the `stop()` method.

@hanishakoneru `omState = State.STOPPED` was set both at the start and end of `stop()` (added in 839fc94e1b and 410e4d3f54, respectively).  Do you recall any specific reason for that, or was it just a harmless duplication?

Reply:
I think it is duplication and can be removed.